cif_pow.dic

```diff
@@ -3248,6 +3291,7 @@ save_PD_INSTR
 ;
     _name.category_id             PD_GROUP
     _name.object_id               PD_INSTR
+    _category_key.name            '_pd_instr.diffractogram_id'
```

Comment:
An instrument description is not tied to a particular diffractogram. The direction is usually the reverse, that a diffractogram implies an instrument.  What is currently implicit is that there is an "instrument id", and each diffractogram is associated with a particular instrument id. We could also go further and associate each detector with a particular instrument id.

Reply:
Fixed